src/delete-branch.ts

```diff
@@ -9,6 +9,16 @@ export default async (options: DeleteBranchOptions) => {
   log.info("Deleting branch from Crowdin");
 
   try {
+    // Delete any open tasks for this branch
+    const tasks = await listTasks();
+
+    // Iterate over tasks and delete any where the task name includes the branch name
+    for await (const task of tasks.data) {
```

Comment:
NBC: Why not use `Promise.allSettled` so this work can happen in parallel? For bonus point we can log any tasks that failed to be deleted by assigning the return value to a variable and seeing if any of the items in the array have a `.status` of `rejected`.